packages/kbn-securitysolution-autocomplete/src/field_value_wildcard/index.tsx

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useCallback, useMemo, useState, useEffect } from 'react';
+import { EuiFormRow, EuiComboBoxOptionOption, EuiComboBox } from '@elastic/eui';
+import { DataViewBase, DataViewFieldBase } from '@kbn/es-query';
+
+import { uniq } from 'lodash';
+
+import { ListOperatorTypeEnum as OperatorTypeEnum } from '@kbn/securitysolution-io-ts-list-types';
+
+// TODO: I have to use any here for now, but once this is available below, we should use the correct types, https://github.com/elastic/kibana/issues/100715
+// import { AutocompleteStart } from '../../../../../../../src/plugins/data/public';
+type AutocompleteStart = any;
+
+import * as i18n from '../translations';
+import { useFieldValueAutocomplete } from '../hooks/use_field_value_autocomplete';
+import {
+  getGenericComboBoxProps,
+  GetGenericComboBoxPropsReturn,
+} from '../get_generic_combo_box_props';
+import { paramIsValid } from '../param_is_valid';
+
+const SINGLE_SELECTION = { asPlainText: true };
+
+interface AutocompleteFieldWildcardProps {
+  placeholder: string;
+  selectedField: DataViewFieldBase | undefined;
+  selectedValue: string | undefined;
+  indexPattern: DataViewBase | undefined;
+  isLoading: boolean;
+  isDisabled: boolean;
+  isClearable: boolean;
+  isRequired?: boolean;
+  fieldInputWidth?: number;
+  rowLabel?: string;
+  autocompleteService: AutocompleteStart;
+  onChange: (arg: string) => void;
+  onError: (arg: boolean) => void;
+  onWarning: (arg: boolean) => void;
+  warning?: string;
+}
+
+export const AutocompleteFieldWildcardComponent: React.FC<AutocompleteFieldWildcardProps> = ({
+  autocompleteService,
+  placeholder,
+  rowLabel,
+  selectedField,
+  selectedValue,
+  indexPattern,
+  isLoading,
+  isDisabled = false,
+  isClearable = false,
+  isRequired = false,
+  fieldInputWidth,
+  onChange,
+  onError,
+  onWarning,
+  warning,
+}): JSX.Element => {
+  const [searchQuery, setSearchQuery] = useState('');
+  const [touched, setIsTouched] = useState(false);
+  const [error, setError] = useState<string | undefined>(undefined);
+  const [isLoadingSuggestions, , suggestions] = useFieldValueAutocomplete({
+    autocompleteService,
+    fieldValue: selectedValue,
+    indexPattern,
+    operatorType: OperatorTypeEnum.WILDCARD,
+    query: searchQuery,
+    selectedField,
+  });
+  const getLabel = useCallback((option: string): string => option, []);
```

Comment:
This is because the method that populates the combo box expects this to be a function. Didn't want to change this as it's used in the other operator components